Gemfile.lock

```diff
@@ -128,19 +128,19 @@ GEM
       activerecord (>= 3.1.0, < 8)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.587.0)
-    aws-sdk-core (3.130.2)
+    aws-partitions (1.588.0)
```

Comment:
Data for aws service regions and endpoints https://my.diffend.io/gems/aws-partitions/1.587.0/1.588.0